auth/auth_test.go

```diff
@@ -46,7 +46,7 @@ func TestMain(m *testing.M) {
 		ctx   context.Context
 		creds *google.DefaultCredentials
 	)
-
+	opt := option.WithCredentialsFile("../testdata/service_account.json")
```

Comment:
I believe this is only meaningful when running without the dev server. Can we move it inside the `else` block below?